openmrs/i18n/registration/locale_fr.json

```diff
@@ -37,9 +37,10 @@
   "ADDRESSHIERARCHY_DISTRICT":"District",
   "REGISTRATION_IDTYPE_1_CNI_KEY":"CNI",
   "REGISTRATION_IDTYPE_2_ART_KEY":"ART",
-  "REGISTRATION_IDTYPE_3_PASSPORT_KEY":"Passport",
-  "REGISTRATION_IDTYPE_4_RECEIPT_KEY":"Récépissé",
-  "REGISTRATION_IDTYPE_5_OTHER_KEY":"Autre",
+  "REGISTRATION_IDTYPE_3_ANC_KEY":"ANC",
```

Comment:
@lauravignoli Could you please add [FR] to CNI, ART and ANC?

Reply:
done :)